api/src/pcapi/connectors/acceslibre.py

```diff
@@ -0,0 +1,148 @@
+"""
+A client for API Acceslibre.
+Documentation of the API: https://acceslibre.beta.gouv.fr/api/docs/
+Further explanations at: https://schema.data.gouv.fr/MTES-MCT/acceslibre-schema/0.0.14/documentation.html
+"""
+
+import json
+import logging
+
+from fuzzywuzzy import fuzz
+from slugify import slugify
+
+from pcapi import settings
+from pcapi.utils import module_loading
+from pcapi.utils import requests
+
+
+logger = logging.getLogger(__name__)
+
+ACCESLIBRE_REQUEST_TIMEOUT = 3
+REQUEST_PAGE_SIZE = 50
+
+
+def _get_backend() -> "BaseBackend":
+    backend_class = module_loading.import_string(settings.ACCESLIBRE_BACKEND)
+    return backend_class()
+
+
+def find_venue_at_accessibility_provider(
+    name: str,
+    public_name: str | None = None,
+    siret: str | None = None,
+    ban_id: str | None = None,
+    city: str | None = None,
+    postal_code: str | None = None,
+) -> str | None:
+    """Try to find the entry at our accessibility provider that matches our venue
+    Returns uuid at provider
+    """
+    return _get_backend().find_venue_at_accessibility_provider(
+        name=name,
+        public_name=public_name,
+        siret=siret,
+        ban_id=ban_id,
+        city=city,
+        postal_code=postal_code,
+    )
+
+
+def match_by_name(results_at_provider: list[dict[str, str]], name: str, public_name: str | None) -> str | None:
+    name = name.lower()
+    public_name = public_name.lower() if public_name else "PUBLIC_NAME_MISSING"
+    for erp in results_at_provider:
+        name_at_provider = erp["nom"].lower()
+        if (
+            name_at_provider in name
+            or name in name_at_provider
+            or public_name in name_at_provider
+            or name_at_provider in public_name
+        ):
+            return erp["uuid"]
+        if (50 < fuzz.ratio(name_at_provider, name) <= 100) or (50 < fuzz.ratio(name_at_provider, public_name) <= 100):
```

Comment:
[micro-détail] La fonction `ratio` peut-elle renvoyer une valeur supérieure à 100 ? Je suppose que non. On pourrait alors légèrement simplifier les conditions : 

```suggestion
        if fuzz.ratio(name_at_provider, name) > 50 or fuzz.ratio(name_at_provider, public_name) > 50:
```